ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/FragmentationServiceImpl.java

```diff
@@ -24,18 +28,18 @@ public FragmentationServiceImpl(LdesConfig ldesConfig, ViewConfig viewConfig,
     @Override
     public LdesFragment getFragment(String collectionName, String path, String value) {
         return ldesFragmentRespository.retrieveFragment(collectionName, path, value)
-                .orElseGet(()-> createDummyFragment(collectionName, path));
+                .orElseGet(()-> createDummyFragment(collectionName, List.of(new FragmentPair(path, value))));
```

Comment:
I'm wondering if 'Dummy' is the right word compared to 'EmptyFragment' for example. But I presume it boils down to taste :)

Reply:
renamed dummy to empty